COLLADAMaya/src/COLLADAMayaPhysXExporter.cpp

```diff
@@ -4577,20 +4578,22 @@ namespace COLLADAMaya
         int curr_a = 0;
         int curr_b = 0;
 
-        if (!ExtractPhysXPluginVersionNumbers(requ_major, requ_minor, requ_a, requ_b)) {
+        if (!ExtractPhysXPluginVersionNumbers(GetRequiredPhysXPluginVersion(), requ_major, requ_minor, requ_a, requ_b)) {
             return false;
         }
 
-        if (!ExtractPhysXPluginVersionNumbers(curr_major, curr_minor, curr_a, curr_b)) {
+        if (!ExtractPhysXPluginVersionNumbers(GetInstalledPhysXPluginVersion(), curr_major, curr_minor, curr_a, curr_b)) {
             return false;
         }
 
-        if (curr_major < requ_major ||
-            curr_minor < requ_minor ||
-            curr_a < requ_a ||
-            curr_b < requ_b) {
-            return false;
-        }
+		if (curr_major > requ_major) return true;
```

Comment:
Are you sure that major plugin changes won't break openCollada plugin?
Usually a change in major require testing and some code change, otherwise only the minor cereion number would be changed. 
So this line should be changed. And only tested against known working major numbers.